tests/functional/test_basic.py

```diff
@@ -162,17 +162,15 @@ def test_email(pyi_builder):
         """)
 
 
-@skipif(is_module_satisfies('Crypto >= 3'), reason='Bytecode encryption is not '
-        'compatible with pycryptodome.')
-@importorskip('Crypto')
+@importorskip('tinyaes')
 def test_feature_crypto(pyi_builder):
     pyi_builder.test_source(
         """
         from pyimod00_crypto_key import key
         from pyimod02_archive import CRYPT_BLOCK_SIZE
 
-        # Issue 1663: Crypto feature caused issues when using PyCrypto module.
-        import Crypto.Cipher.AES
+        # An issue similar to the 1663 with PyCrypto may arise for tinyaes
```

Comment:
I'd remove the comment unless you have any reason to believe that the issue could happen.

Reply:
I had a look on the issue #1663, and the test is there to check the the module namespace is cleaned up after loading the AES module during the bootstrap.

It seems to be legit, what do you think if I add some more explicit info instead of a plain issue number?

Reply:
@naufraghi good point on the legit - I reread it and agree now. Just add a bit more info in the comment.

Reply:
Reworded a bit